templates/competitorNotFound.html

```diff
@@ -1,5 +1,6 @@
 <template name="participantNotFound">
```

Comment:
Ohhh, for consistency's sake, would you rename this file from competitorNotFound to participantNotFound?

Lets do that with every file that still has competitor in the name, please (see https://github.com/cubing/ccm/pull/175 for where we did this). Here's what I see in my clone:

```
~/gitting/ccm @kaladin> git ls-files | grep -i competitor
client/advanceCompetitors.js
client/competitorResults.js
css/advanceCompetitors.import.less
templates/advanceCompetitors.html
templates/competitorNotFound.html
templates/competitorResults.html
~/gitting/ccm @kaladin> 
```